proto/cosmos/crypto/ed25519/keys.proto

```diff
@@ -29,7 +29,7 @@ message PubKey {
   bytes key = 1 [(gogoproto.casttype) = "crypto/ed25519.PublicKey"];
 }
 
-// Deprecated: PrivKey defines a ed25519 private key.
```

Comment:
not sure why this is deprecated

Reply:
I think we wanted to use tendermint type back in the time.